01week/rockPaperScissors.js

```diff
@@ -9,11 +9,32 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1=hand1.toLowerCase();
+  hand2=hand2.toLowerCase();
+  hand1=(hand1.trim());
```

Comment:
These two methods can be combined together on the same line.  .toLowerCase().trim()